carma_clock_py/src/carma_clock_py.cpp

```diff
@@ -0,0 +1,17 @@
+#include <pybind11/pybind11.h>
+
+#include <carma_clock.h>
+namespace py = pybind11;
+
+PYBIND11_MODULE(carma_clock_py, m) {
```

Comment:
Just a suggestion. Maybe rename this to something like "carma_clock_cpp_to_python_adapter"?